.github/workflows/semver.yml

```diff
@@ -15,13 +15,54 @@ jobs:
           fetch-depth: 0
       - name: Get latest gitwerk
         run: |
-          wget "https://github.com/rinx/gitwerk/releases/latest/download/gitwerk-linux-amd64.zip"
+          wget "https://github.com/rinx/gitwerk/releases/download/v0.1.2/gitwerk-linux-amd64.zip"
           unzip gitwerk-linux-amd64.zip
       - name: Run gitwerk semver-auto
+        id: semver
         run: |
-          ./gitwerk semver-auto
+          GITWERK_RESULT=`./gitwerk sci << EOF
+          (let [ctx {:options {:dry-run? true}}
+                res (semver-auto ctx nil)
+                status (get-in res [:console-out :status])
+                oldv (get-in res [:console-out :old-version])
+                newv (get-in res [:console-out :new-version])]
+            (if (= status :updated)
+              newv
+              "not-updated"))
+          EOF`
```

Comment:
![badge](https://img.shields.io/badge/review-note-informational.svg?style=flat-square)
This is a tiny Clojure script to decide whether the current commit should be versioned or not. It returns an updated version name or "not-updated" string.